apps/web/src/routes/_app/admin/users/create.tsx

```diff
@@ -155,6 +156,19 @@ const RouteComponent = () => {
                 path: ['confirmPassword']
               });
             }
+
+            const existingUsers = await axios.get(`/v1/users`);
```

Comment:
Do not use async validation schema. This can cause errors if the libui code is refactored and the schema is not verified async. Instead, do this request in the submit handler. Return the message and success false if an issue.